Neural Networks/01.MNIST Classification/utils.py

```diff
@@ -174,16 +177,89 @@ def train_model(mdl, images, labels, val_images, val_labels, num_e, batch_size,
                           callbacks=call_backs)
     return history
 
+
 # evaluate the model
 def evaluate_model(mdl, images, labels):
-    '''
+    """
     This function evaluates the model on the `scaled_test_images` and `test_labels`.
     and returns a tuple (test_loss, test_accuracy).
 
     :param mdl: Sequential model from 'get_model'
     :param images: numpy.ndarray (n_test, pixel, pixel, channel=1)
     :param labels: numpy.ndarray (n_test,1)
     :return: tuple (test_loss, test_accuracy)
-    '''
+    """
     test_loss, test_accuracy = mdl.evaluate(images, labels)
     return test_loss, test_accuracy
+
+
+# image cropping
+def remove_image_margins(image):
```

Comment:
This function is not very efficient or pythonic. You should use `np.where`. Specifically, `np.where(image > 0)` will give you the y, x pixels corresponding to non-background. You can then find min and maxes of those.